spec/definitions.yaml

```diff
@@ -683,9 +683,10 @@ definitions:
         type: boolean
         x-nullable: false
         description: |
-          If true, the version is available for new clusters and cluster
+          If `true`, the version is recommended for use in cluster creation and cluster
           upgrades. Older versions become unavailable and thus have the
-          value `false` here.
+          value `false` here. Cluster creation with this version is still possible
```

Comment:
"still possible", but under the condition that there is at least one cluster running with the given release version in the installation.

Reply:
That’s not how it’s currently implemented. Cluster creation is possible with every release in the CP